modules/nw-pod-network-connectivity-checks.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+//
+
+[id="nw-pod-network-connectivity-checks_{context}"]
+= Connection health checks performed
+
+To verify what cluster resources are reachable, the following TCP connection tests are performed for the cluster API services:
+
```

Comment:
Do we need to define what these tests are and what failures would indicate? Or will folks who know this stuff know that suff?

Reply:
@mburke5678, oops, yeah, what I wrote presents the appearance of different kinds of tests happening. It's actually just a single TCP connection test, which is to say an attempt to connect to the service.

I'll try a different formulation.